src/components/rating/rating.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
```

Comment:
UX Suggestions: 
1. pointer cursor on hover could be added.
![image](https://user-images.githubusercontent.com/91944618/154858092-f809d419-9cc7-42a9-a66e-5c2dd63fac11.png)